TAC/Meetings/2025/2025-01-23/CCC-TAC-Minutes-2025-01-23.md

```diff
@@ -0,0 +1,167 @@
+*
+
+## Old Business
+
+* DM reviewed the old Business covered in the previous meeting
+
+
+### Announcements 
+* Mentorship: Yash Mankand (YM) shared the details of the mentees and extended gratitude to the mentors for their efforts. 
+* DM shared that the 2025 Tech Talk Schedule is open and welcomes more topics feel free to reach out to Fritz Alder (FA). MB proposed the idea of organizing webinars which can be hosted on the CCC website along with Tech talks to spread the word.  
+* Annual project updates: DM shared that the annual project schedule will be shared in early 2025.
+
+## Antitrust Policy & Inclusiveness Trainings
+* RK briefed about the available antitrust policy and Inclusiveness trainings. MB reiterated the importance of the shared training. DM emphasized that the training shall be covered by the TAC members.
+
+## Discussion:   Workload Identity 
+Mark Novak (MN) led the discussion on the workload identity. MN shared that he has already shared the details in the email (for reference). MN proposed to have dedicated people from the TAC members (for example having a separate SIG) for the workload identity for the success of Confidential Computing in the marketplace. Alec Fernandez (AF) appreciated the use cases shared in a link by YM. 
+MN emphasized that a stronger collaboration is required. MB mentioned discussing this topic in the GB meeting too. 
+
+## Project Proposal: OpenVMM 
+Caroline Perez-Vargas (CPV) presented the OpenVMM project to the TAC members. 
+Members of the TAC asked clarifying questions about the project, which CPV answered.
+
+
+DM asked about the MS Open Source Code of Conduct. CPV shall look into the differences between the Contributor Covenent and the Microsoft code of conduct 
+
+DM asked about copy left implications. CPV said they do not foresee implications.
+
+MB asked about the existing community and how the project would extend the community and if there are other contributors then MS? CPV the project already had external contributions, however most is still done by MS. They are all looking at ways to socialize the project. Looking for more partners and having public meetings. They are also looking for guidance from the CCC on how to reach these goals.
+
+DM ask if their is a way for contributors to become maintainers and if the project are interested in external maintainers. CPV said they plan for it, but would develop the plan with the community rather than use their own internal arrangements.
+
+
+The motion to accept the project proposal was made by Alec Fernandez and seconded by  Paul Howard.
+
+* Paul Howard - ARM - Yes
+* Intel - Dan Middleton (TAC chair) - Yes
+* Microsoft - Alec Fernandez - Yes 
+* Red Hat - Yash Mankad - Yes
+* Fritz Alder (NVIDIA) - Yes
+
+The project has been approved by the TAC. The voting members that were not in the TAC meeting would also be offered the opportunity to vote by email. The project proposal will be taken to the Governing Board meeting as well for approval.
```

Comment:
```suggestion
The project has been approved by the TAC. The voting members that were not present will also be offered the opportunity to vote by email. The project proposal will be taken to the Governing Board meeting as well for approval.
```
Please add to the minutes the email votes of the absent members (noting that they were sent by email).